tests/phpunit/Civi/Funding/Permission/ContactRelation/Loader/RelationshipLoaderTest.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/*
+ * Copyright (C) 2024 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+declare(strict_types = 1);
+
+namespace Civi\Funding\Permission\ContactRelation\Loader;
+
+use Civi\Api4\Relationship;
+use Civi\Api4\RelationshipType;
+use Civi\Funding\AbstractFundingHeadlessTestCase;
+use Civi\Funding\Fixtures\ContactFixture;
+use Civi\Funding\Fixtures\ContactTypeFixture;
+use Civi\Funding\Fixtures\GroupContactFixture;
+use Civi\Funding\Fixtures\GroupFixture;
+use Civi\PHPUnit\Traits\ArrayAssertTrait;
+use Civi\RemoteTools\Api4\Api4;
+
+/**
+ * @covers \Civi\Funding\Permission\ContactRelation\Loader\RelationshipLoader
+ *
+ * @group headless
+ */
+final class RelationshipLoaderTest extends AbstractFundingHeadlessTestCase {
+
+  use ArrayAssertTrait;
+
+  private RelationshipLoader $loader;
+
+  protected function setUp(): void {
+    parent::setUp();
+    $this->loader = new RelationshipLoader(Api4::getInstance());
+  }
+
+  public function testGetContacts(): void {
```

Comment:
testGetContacts => testContactRelations ?

Reply:
It tests the method `getContacts`so the name is fine for me.